### PR TITLE
Automated Testing SA

### PR DESCRIPTION
**Pull request by chasestubb**

### CODE REVIEW

File: pymars/tests/test_create_limbo.py [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    ep_star = 0.5
+    print("---  safe: H, O, OH")
+    safe = ["H", "O", "OH"]
+
+    output = create_limbo(solution_object, ep_star, dic, safe)
+
+    print("---create_limbo <test 1> output :: ") 
+    print(output)
+
+    assert "H" not in output
```
Comment (xMestas) [inline pymars/tests/test_create_limbo.py:87]: Can you also assert that the length of output is 3?

File: pymars/tests/test_create_limbo.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    ep_star = 0.5
+    print("---  safe: H2, O2, H2O")
+    safe = ["H2", "O2", "H2O"]
+
+    output = create_limbo(solution_object, ep_star, dic, safe)
+
+    print("---create_limbo <test 2> output :: ") 
+    print(output)
+
+    assert "H2" not in output
```
Comment (xMestas) [inline pymars/tests/test_create_limbo.py:132]: Same.  You can just say len(output) to get the length

File: pymars/tests/test_create_limbo.py [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+
+    # variable conditions
+    print("---  ep_star: 0.05")
+    ep_star = 0.05
+    print("---  safe: empty list")
+    safe = []
+
+    output = create_limbo(solution_object, ep_star, dic, safe)
+
+    assert "H2" not in output
```
Comment (xMestas) [inline pymars/tests/test_create_limbo.py:172]: Assert length

File: pymars/tests/test_create_limbo.py [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+    ep_star = 1
+    print("---  safe: empty list")
+    safe = []
+
+    output = create_limbo(solution_object, ep_star, dic, safe)
+
+    print("---create_limbo <test 4> output :: ") 
+    print(output)
+
+    assert "H2" in output
```
Comment (xMestas) [inline pymars/tests/test_create_limbo.py:215]: Assert length

File: pymars/tests/test_dic_lowest.py [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+########
+def test_good_input_3():
+    print("---dic_lowest <test 6> input :: good input 3. should be Caius")
+
+    dic = {"Caius": 1, "BC": 9, "CD": 8, "DE": 5} 
+    output = dic_lowest(dic)
+
+    print("---dic_lowest <test 6> output :: " + output) 
+
+    assert output == "Caius"
```
Comment (xMestas) [inline pymars/tests/test_dic_lowest.py]: Keyus

Comment (chasestubb) [reply pymars/tests/test_dic_lowest.py]: will do, captain

**Pull request merged**